### ISSUE TITLE
Rule Engine: copied items add duplicate recommended items

### ISSUE THREAD

**Issue by githubbob42** [githubbob42/mingle2github2#3166]
**[Mingle Card: 3469](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/3469)**
Steps to reproduce
------------------

```
| | |
|-|-|
|**Version #**|
|Beta 2799|
|**OS**|
|Any|
|**Browser**|
|Any|
|**Username**|
|ajf@t3.com|
|**Password**|
|computer47|

  

 


```

1.  Create a ticket on 'Console Rule Engine Job' with Rule Recommended and Rule Required Items
2.  Create a copy of the ticket

Expected result
---------------

The copy and original has identical items

Actual Result
-------------

Copy of the original ticket has duplicate recommended items

Analysis
--------

This is entirely controlled by the recommend-item module.  It checks only to see if the item is recommended.  Instead it would simply have to check that the item exists the same way we do it for recommended.

This would require us to remove a line of code.  The question is do we want it to include required items when it is comparing the ticket?

In other words, if the same item is also required on the ticket, does that satisfy recommended?

For now, we will make this scenario work and not worry about the interaction with required.

Related Cards
-------------

Test Plan
---------

### Checklist for Merging

*   How is this tested in Selenium? 
*   Do we have adequate unit testing? 
*   Where is this covered in documentation?
*   Where is this in Tarantula? 
    *   **Test Case**: Copy Ticket with Rules

**Comment by githubbob42** [comment]
### Related Story: #2664 Rule Engine - Ticket Copy ([Mingle](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/2944))

**Comment by githubbob42** [comment]
**Pull Request #1507**
